src/Surfnet/StepupSelfService/SelfServiceBundle/Service/YubikeySecondFactor/ProofOfPossessionResult.php

```diff
@@ -91,7 +132,7 @@ public function didOtpVerificationFail()
     /**
      * @return boolean
      */
-    public function didProofOfPossessionFail()
+    public function didProofOfPossessionCommandFail()
```

Comment:
It was/is the structure and apparent intent of the Result objects to communicate success/failure and if failed, why it failed. This in order to be able to provide correct feedback to the user (at least, that is what I gathered from it).
This name was changed in order to better reflect the fact that the Proof of Possession as given by the User was in fact successful, however there was a system error (see the named constructor and inner logic of the class). 
I'm not to keen on how these classes are implemented in general though...
Will add a chore to revisit them later as we have more pressing matters to tend to :smiley: